platform/pulpcore/tasking/celery_instance.py

```diff
@@ -13,16 +13,16 @@
 
 DEDICATED_QUEUE_EXCHANGE = 'C.dq2'
 RESOURCE_MANAGER_QUEUE = 'resource_manager'
-CELERYBEAT_SCHEDULE = {
+beat_schedule = {
```

Comment:
Can this be changed back? This is the Pulp constant and should follow the pulp constants naming scheme